source/client/output_formatter_impl.cc

```diff
@@ -189,7 +190,7 @@ DottedStringOutputFormatterImpl::formatProto(const nighthawk::client::Output& ou
               "{}.microseconds: {}", percentile_prefix,
               Envoy::Protobuf::util::TimeUtil::DurationToMicroseconds(percentile.duration()));
         } else {
-          ss << fmt::format("{}.value: {}", percentile_prefix, percentile.raw_value());
+          ss << fmt::format("{}.value: {:.{}f}", percentile_prefix, percentile.raw_value(), 0);
```

Comment:
Not sure I follow this, what is the ", 0" for ?

Reply:
Basically the intent is to avoid trailing `.0` by specifying zero precision. However, I changed this to use a static cast to an integer instead, which I'm hoping is more intuitive?